sdk/core/core-paging/src/getPagedAsyncIterator.ts

```diff
@@ -29,19 +29,19 @@ export function getPagedAsyncIterator<
     byPage:
       pagedResult?.byPage ??
       ((settings?: PageSettings) => {
-        return getPageAsyncIterator(
-          pagedResult as PagedResult<TPage, PageSettings, TLink>,
-          settings?.maxPageSize
-        );
+        const { continuationToken, maxPageSize } = settings ?? {};
+        return getPageAsyncIterator(pagedResult as PagedResult<TPage, PageSettings, TLink>, {
+          continuationToken: continuationToken as unknown as TLink | undefined,
```

Comment:
since continuation tokens might not be a URL, but an opaque token, should we give this a better name like `pageLink`?

Reply:
Good point, I am already using the term link everywhere in the implementation and there is no point to diverge here.